service/src/grandpa_support.rs

```diff
@@ -96,7 +96,7 @@ impl<Block, B> grandpa::VotingRule<Block, B> for PauseAfterBlockFor<NumberFor<Bl
 /// intermediary pending changes are replaced with a static list comprised of
 /// w3f validators and randomly selected validators from the latest session (at
 /// #1500988).
-pub(crate) fn kusama_hard_forks() -> Vec<(
+pub fn kusama_hard_forks() -> Vec<(
```

Comment:
This should also not be required for tests.

Reply:
I'm not sure how I am going to fix this as it is inside the macro :thinking: I will think of a way...

Reply:
done